src/main/scala/com/microsoft/hyperspace/index/sources/FileBasedSourceProviderManager.scala

```diff
@@ -45,28 +44,15 @@ class FileBasedSourceProviderManager(spark: SparkSession) {
     })
 
   /**
-   * Runs createRelation() for each provider.
-   *
-   * @param logicalRelation Logical relation to create [[Relation]] from.
-   * @param fileIdTracker [[FileIdTracker]] to use when populating the data of [[Relation]].
-   * @return [[Relation]] created from the given logical relation.
-   * @throws HyperspaceException if multiple providers returns [[Some]] or
-   *                             if no providers return [[Some]].
-   */
-  def createRelation(logicalRelation: LogicalRelation, fileIdTracker: FileIdTracker): Relation = {
-    run(p => p.createRelation(logicalRelation, fileIdTracker))
-  }
-
-  /**
-   * Runs refreshRelation() for each provider.
+   * Runs refreshRelationMetadata() for each provider.
    *
    * @param relation [[Relation]] to refresh.
    * @return Refreshed [[Relation]].
-   * @throws HyperspaceException if multiple providers returns [[Some]] or
-   *                             if no providers return [[Some]].
+   * @throws [[HyperspaceException]] if multiple providers returns [[Some]] oe
+   *         if no providers return [[Some]].
    */
-  def refreshRelation(relation: Relation): Relation = {
-    run(p => p.refreshRelation(relation))
+  def refreshRelationMetadata(relation: Relation): Relation = {
```

Comment:
We have so many "Relation"s, so I am explicitly calling it out as relation metadata here now.